draft-fossati-tls-attestation.md

```diff
@@ -444,15 +456,16 @@ binder_collection = {
 {: #figure-tls-binder title="Format of TLS Binder Collection."}
```

Comment:
quibbling alert: "collection" does not sound entirely right.  maybe "attestation channel binder"?